Description: Correct pod NAME issues in LedgerSMB/DBObject/Asset_Report.pm
 Has and incorrect module name and a comma instead of a dash in the NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472554&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/DBObject/Asset_Report.pm
+++ b/LedgerSMB/DBObject/Asset_Report.pm
@@ -3,7 +3,7 @@
 
 =head1 NAME
 
-LedgerSMB::DBObject::Asset_Report.pm, LedgerSMB Base Class for Asset Reports
+LedgerSMB::DBObject::Asset_Report - LedgerSMB Base Class for Asset Reports
 
 =head1 SYNOPSIS
 
